readme.md

```diff
@@ -65,9 +78,14 @@ A normalized string representing the entire address that was passed into the con
 ### Installing dependencies
 
 1. install only required dependencies to run sweeper
-   - `pip install -e .`
+    - `pip install -e .`
 1. install required dependencies to work on sweeper
-   - `pip install -e ".[develop]"`
+    - `pip install -e ".[develop]"`
 1. install required dependencies to run sweeper tests
-   - `pip install -e ".[test]"`
+    - `pip install -e ".[test]"`
 1. run tests: `pytest`
+
+### Uploading to pypi.org
+
+1. `python setup.py sdist`
+1. `twine upload dist/*`
```

Comment:
what is twine?

Reply:
The recommended way of uploading to pypi now. If you try to do the old way `python setup.py upload` it fails and points you to twine.

Reply:
is twine something i would pip install or what is that?

Reply:
the twine package shows `python setup.py sdist bdist_wheel`. Did you try creating a wheel? I remember reading they are better than the original...

Reply:
I see that in the docs now. It wasn't in the tutorial I was following. I built a wheel and uploaded it as well for v1.0.3 so now it looks like this:
![image](https://user-images.githubusercontent.com/1326248/75045050-d509f200-547f-11ea-8dc8-7f0ba480f961.png)

I'll update the readme...